### ISSUE TITLE
Constant prediction parameters

### ISSUE THREAD

**Issue by mengyaoji** [StatMixedML/XGBoostLSS#23]
When I try to use BCT distribution for a dataset, the resulting prediction parameters (location, nu, tau) are all constant. I believe nu, tar are their initial values (0.5,10). Only scale changes like the example with Gaussian distribution. I am wondering what does this mean? 

The parameters I used for training are:
`
Best trial:
  Value: 125.35836839999999
  Params: 
    eta: 0.040946512023655214
    max_depth: 4
    gamma: 2.9779429635527073e-05
    subsample: 0.2887880027234064
    colsample_bytree: 0.3208686624698766
    min_child_weight: 316
    opt_rounds: 500
`

**Comment by StatMixedML** [comment]
Dear @mengyaoji,

thanks for the interest in the project.

What you describe is indeed an odd behaviour. The hyper-parameters don't seem to be too odd. It is difficult to say what the problem is. Can you maybe provide a minimum working example with a data snippet?

**Comment by mengyaoji** [comment]
Many thanks for your prompt reply. A bit more information, I tried to scale y using y/100. The location is now no longer constant. But nu and tau are still the initial values. My data is quite large, I will try to see if I can reduce the data and build a minimum working example. One thing special for my data is I have many more features than the number of samples. The data have ~5000 columns but only ~200 rows. 

At the same time, do you have a working example for BCT distribution. That would also be helpful for me to better understand the problem.

**Comment by StatMixedML** [comment]
@mengyaoji A minimum working example would be great. We can use this also as an example how to use the distribution. You don't need to provide the full dataset with all columns, but a subset would be enough. Important is that the data has full rows.

**Comment by mengyaoji** [comment]
Unfortunately, the data I am using now cannot be shared. Let me find a similar dataset to reproduce the problem. Thanks for your help.

**Comment by StatMixedML** [comment]
@mengyaoji You may also want to try using a normalization `distribution.stabilize = "MAD"`. Since XGBoostLSS updates the parameter estimates by optimizing Gradients and Hessians, it is important that these are comparable in magnitude for all distributional parameters. Due to variability regarding the ranges, the estimation of Gradients and Hessians might become unstable so that XGBoostLSS might not converge or might converge very slowly.  This is why you might see constant predictions for some of the parameters.

**Comment by StatMixedML** [comment]
@mengyaoji Kindly asking if there is any update on this you can share?
